hashednode.go

```diff
@@ -31,14 +31,37 @@ import (
 )
 
 type HashedNode struct {
-	commitment  []byte
+	commitment  SerializedPointCompressed
+	serialized  SerializedPoint
```

Comment:
`serialized` is a new field, which is exactly the serialized representation that the DB has.
Now this is our source of truth for `HashedNode`. We _always_ have value in this field.

As in, when we want to create a _HashedNode_, we create it with the serialized bytes we got from the database. No CPU work, just storing.